web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1934,3 +1966,174 @@ form#views-exposed-form-job-board-page {
     margin-left: 0px;
   }
 }
+
+.gm-style .gm-style-iw-c {
+  position: absolute;
+  overflow: hidden;
+  top: 0;
+  left: 0;
+  transform: translate(-50%, -100%);
+  background-color: rgba(42, 85, 107, 0.85);
+  border: 2px solid #FF9049;
+  border-radius: 0px;
+  padding-left: 20px;
+  padding-right: 20px !important;
+  padding-bottom: 12px !important;
+  box-shadow: none !important;
+}
+
+.gm-style .gm-style-iw {
+  font-family: Lato;
+  font-weight: 300;
+  font-size: 13px;
+}
+
+.gm-style .gm-style-iw h2 a {
+  font-size: 14px;
+  margin: 8px 0 6px 0;
+  color: #FFFFFF;
+
+  &:hover {
+    color: #569bdd;
+  }
+}
+
+.gm-style .gm-style-iw h2 {
+  font-size: 14px;
+  margin: 8px 0 6px 0;
+  color: #FFFFFF;
+  text-transform: uppercase;
+
+  &:hover {
+    color: #569bdd;
+  }
+}
+
+.gm-style .gm-style-iw h2, .h2 {
+  line-height: 1em;
+  padding-bottom: 12px;
+  border-bottom: 1px solid #FF9049;
+  margin-right: 16px;
+}
+
+@media (max-width: $md) {
+  .gm-style .gm-style-iw h2, .h2 {
+    margin-right: 0px;
+  }
+}
+
+.gm-style .gm-style-iw p {
+  padding-left: 0px;
+  padding-right: 20px;
+  color: #FFFFFF;
+}
+
+.gm-style .gm-style-iw .row {
+  margin-left: 0px;
+  margin-right: 0px;
+}
+
+.gm-style .gm-style-iw p.address {
+  color: #FFFFFF;
+}
+
+.gm-style .gm-style-iw-t::before {
+  background: linear-gradient(45deg,rgba(255, 144, 73, 1) 50%,rgba(255,255,255,0) 51%,rgba(255,255,255,0) 100%);
+  content: "";
+  height: 30px;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transform: translate(-50%,-50%) rotate(-45deg);
+  width: 30px;
+}
+
+.gm-style .gm-style-iw-t::after {
+  visibility: hidden;
+}
+
+.gm-style .gm-style-iw div:first-child {
+  padding: 0px;
+}
+
+.company-info-container-map .field-label {
+  font-size: 1em;
+  font-family: Montserrat;
+  font-weight: 800;
+  color: #FF9049;
+}
+
+.company-info-container-map a {
+  color: #FFFFFF;
+
+  &:hover {
+    color: #569BDD;
+  }
+}
+
+.gm-style-iw-d {
+  overflow: hidden !important;
+}
+
+.gm-style .gm-style-iw p:first-child {
+  padding-right: 20px;
+}
+
+.distance-container span.field-prefix {
```

Comment:
I think in general we could group these together better. This .distance-container is to do with the filter at the top of the page, but the code is with stuff about the map popup. this should go near the other code to do with that filter.